internal/querycoord/task_scheduler.go

```diff
@@ -393,7 +393,7 @@ func (scheduler *TaskScheduler) unmarshalTask(taskID UniqueID, t string) (task,
 		if err != nil {
 			return nil, err
 		}
-		fullReq, err := generateFullWatchDmChannelsRequest(scheduler.broker, &req)
+		fullReq, err := generateFullWatchDmChannelsRequest(scheduler.ctx, scheduler.broker, &req)
 		if err != nil {
```

Comment:
`scheduler.ctx` or `task.traceCtx()`?

Reply:
nice catch, I think task.traceCtx() is better